Documentation/cli-installation-scenarios.md

```diff
@@ -0,0 +1,266 @@
+| Production          	| Actual releases. Most users are encouraged to install these.                                                                                                                                                                                                      	|
+Below table shows the mapping between the channels, branches and feeds for the Debian pacakage.
+
+| Channel    	| Branch    	| Debian feed 	| Debian package name 	|
+|------------	|-----------	|-------------	|---------------------	|
+| Nightly    	| master    	| Development 	| dotnet-nightly      	|
+| Preview    	| rel/<ver> 	| Development 	| dotnet              	|
+| Production 	| rel/<ver> 	| Production  	| dotnet              	|
+
+Channels also impact the NuGet packages. Based on the version of the package specified, the user may end up getting different channels. The table below shows examples of that.
+
+| NuGet package version 	| Channel            	|
+|-----------------------	|--------------------	|
+| 1.0.0-dev-*           	| Dev channel        	|
+| 1.0.0-beta-*          	| Preview channel    	|
+| 1.0.0                 	| Production channel 	|
+
+# Funnels and discovery mechanisms for CLI bits
+There are multiple ways that we will funnel users towards the installers for the CLI:
+
+1. [Getting Started Page](https://aka.ms/dotnetcoregs)
+2. [Repo landing page](https://github.com/dotnet/cli/blob/rel/1.0.0/README.md)
+3. Package repositories for platforms (`apt-get`, `brew` etc.)
+4. IDEs and editors that integrate with CLI (e.g. Visual Studio, VS Code, Sublime etc.)
+
+Out of the above, the first two funnels are under the control of the CLI team so we will go into slightly more details. The rest of the funnels will use a prescribed way to get to the bits and will have guidance on what bits to use.  
+
+## Getting Started page
+The page can be found on https://aka.ms/dotnetcoregs. This is the main curated first-run funnel for the dotnet CLI. The intent of the page is to help users test out the CLI quickly and become familiar with what the platform offers. This should be the most stable and curated experience we can offer. 
+
+The Getting Started page should only point users to curated install experiences that can contain only stable or LKG bits. 
+
+The below table shows other pertinent information for installs on the "Getting started" page. 
+ 
+| Property              	| Description                                                  	|
+|-----------------------	|--------------------------------------------------------------	|
+| Debian feed           	| Development                                                  	|
+| Brew repo/tap           	| Brew binary repo (https://github.com/Homebrew/homebrew-binary)|
+| CentOS feed               | TBD
+| Local install scripts 	| Latest from rel/1.0.0                                        	|
+
```

Comment:
Getting Started page should not have a NuGet feed methinks. That is too much info. Repo Landing page should take that info.